ocp4/profiles/cis.profile

```diff
@@ -13,6 +13,8 @@ description: |-
 
     Note that this part of the profile is meant to run on the Platform that
     Red Hat OpenShift Container Platform 4 runs on top of.
+
+    The applicable version of OpenShift that this profile addresses is 4.6.
```

Comment:
We've traditionally stayed away from minor version tracking upstream since it not only adds a large maintenance burden, but usually, downstream is responsible for maintaining the minor version.... however, that is in a traditional RHEL sense. Also, upstream traditionally lags behind or way ahead of a downstream release. Does it make sense to handle minor versions?

Reply:
uhm... Maybe what we really want to explain to folks is that this profiles are for 4.6+ and won't be applicable for 4.5 and lower releases.

Reply:
okay. That makes sense. How about something like:

`This profile is applicable to OpenShift versions 4.6 and greater.`